dashboard/src/inspect.ts

```diff
@@ -0,0 +1,127 @@
+export class WebsocketApi {
+    private name:string = "proxyInspect"
```

Comment:
Sorry, this file might be confusing :)  but `WebsocketApi` is core to the `ProxyDashboard` in the sense that it provides a simple interface to communicate with `dashboard.py`.  It implements all client side logic for every API provided by `dashboard.py` over websockets.

This file doesn't tie to `proxyInspect` tab actually.  Yes, it does provide a `enable_inspection` / `disable_inspection` commands, but they are just methods exposed by the backend.  Example, `WebsocketApi` file will not be responsible for painting the `Inspect Traffic` tab.

Lets create a separate file for inspect traffic tab :)